typescript/infra/config/environments/mainnet2/core.ts

```diff
@@ -13,6 +13,8 @@ export const core: ChainMap<MainnetChains, CoreConfig> = {
         '0x3bc014bafa43f93d534aed34f750997cdffcf007',
         '0xd79d506d741fa735938f7b7847a926e34a6fe6b0',
         '0xe4a258bc61e65914c2a477b2a8a433ab4ebdf44b',
+        '0x6aea63b0be4679c1385c26a92a3ff8aa6a8379f2', // staked
```

Comment:
Do you think a 3/7 is the right config? Maybe instead a 4/6 and remove an Abacus Works validator?

Reply:
Indeed